tests/data_context/cloud_data_context/test_include_rendered_content.py

```diff
@@ -0,0 +1,170 @@
+    data_context_fixture_name: str,
+    request,
+) -> None:
+    """
+    All Cloud-backed contexts (DataContext, BaseDataContext, and CloudDataContext) should save an ExpectationSuite
+    with rendered_content by default.
+    """
+    context = request.getfixturevalue(data_context_fixture_name)
+
+    with mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend.list_keys"
+    ), mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend._set"
+    ):
+        expectation_suite: ExpectationSuite = context.create_expectation_suite(
+            "test_suite"
+        )
+    expectation_suite.expectations.append(
+        ExpectationConfiguration(
+            expectation_type="expect_table_row_count_to_equal", kwargs={"value": 10}
+        )
+    )
+    assert expectation_suite.expectations[0].rendered_content is None
+
+    with mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend.list_keys"
+    ), mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend._update"
+    ) as mock_update:
+        context.save_expectation_suite(
+            expectation_suite,
+        )
+
+        # remove dynamic great_expectations version
+        mock_update.call_args[1]["value"].pop("meta")
+
+        mock_update.assert_called_with(
+            ge_cloud_id="None",
+            value={
+                "expectations": [
+                    {
+                        "meta": {},
+                        "kwargs": {"value": 10},
+                        "expectation_type": "expect_table_row_count_to_equal",
+                        "rendered_content": [
+                            {
+                                "value": {
+                                    "schema": {
+                                        "type": "com.superconductive.rendered.string"
+                                    },
+                                    "params": {
+                                        "value": {
+                                            "schema": {"type": "number"},
+                                            "value": 10,
+                                        }
+                                    },
+                                    "template": "Must have exactly $value rows.",
+                                    "header": None,
+                                },
+                                "name": "atomic.prescriptive.summary",
+                                "value_type": "StringValueType",
+                            }
+                        ],
+                    }
+                ],
+                "ge_cloud_id": None,
+                "data_asset_type": None,
+                "expectation_suite_name": "test_suite",
+            },
+        )
+
+
+# TODO: ACB - Enable this test after merging fixes in PRs 5778 and 5763
```

Comment:
This test should be uncommented in either: https://github.com/great-expectations/great_expectations/pull/5778 or https://github.com/great-expectations/great_expectations/pull/5763

Reply:
@anthonyburdi ^

Reply:
Update: remove `xfail` instead of uncomment.